aws-ts-vpc-with-ecs-fargate-py/ecs-fargate-python/__main__.py

```diff
@@ -0,0 +1,173 @@
+"""An AWS fargate with crosswalk vpc"""
+
+import pulumi
+from pulumi import export, ResourceOptions, Config, StackReference, get_stack, get_project
+import json
+import pulumi_aws as aws
+
+#https://www.pulumi.com/docs/intro/concepts/organizing-stacks-projects/#inter-stack-dependencies
+
+# read local config settings - network
+config = Config()
+
+# reading in vpc StackReference Path from local config
+mystackpath = config.get("mystackpath")
```

Comment:
```suggestion
mystackpath = config.require("mystackpath")
```

Reply:
Fixed

Reply:
Looks like this is still `.get` when I suggested `.require`. Do you need to push an updated branch?